BootloaderCommonPkg/Library/PartitionLib/PartitionLib.c

```diff
@@ -485,7 +485,7 @@ GetLogicalPartitionInfo (
     return EFI_INVALID_PARAMETER;
   }
 
-  if (SwPart >= PartBlockDev->BlockDeviceCount) {
+  if ( (SwPart >= PartBlockDev->BlockDeviceCount)||(SwPart >= PART_MAX_BLOCK_DEVICE) ) {

```

Comment:
Small request, can you fix the spacing? See comment from Aiden here:

https://github.com/slimbootloader/slimbootloader/pull/929/files#r543716583